pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -907,6 +907,28 @@ private void charge4MenuItem_Click(object sender, EventArgs e)
         {
             _graphSpectrumSettings.ShowCharge4 = !_graphSpectrumSettings.ShowCharge4;
         }
+        private void synchMzScaleToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            if (_graphFullScan == null || !_graphFullScan.Visible || _graphSpectrum == null || !_graphSpectrum.Visible) return;
+            Settings.Default.SyncMZScale = synchMzScaleToolStripMenuItem.Checked;
+            IMzScaleCopyable source = (IMzScaleCopyable)(synchMzScaleToolStripMenuItem.Owner as ContextMenuStrip)?.SourceControl?.Parent?.Parent;
+
+            if (sender is IMzScaleCopyable)         //testing support
+                source = (IMzScaleCopyable) sender;
+            if (source == null) return;
+            IMzScaleCopyable target = _graphSpectrum;
+            if (source is GraphSpectrum)
+                target = _graphFullScan;
+            target?.SetMzScale(source.Range);
+        }
+
+        public void SynchMzScale(bool direction = false)
+        {
+            if(direction)
+                synchMzScaleToolStripMenuItem_Click(_graphSpectrum, EventArgs.Empty);

```

Comment:
We don't use "if(x)" but "if (x)" instead. We don't call event handler methods directly, but introduce a function that the event handler calls and call that from the other function.

Reply:
Corrected.